diff --git a/lwadvapi/include/public/krb5error-table.h b/lwadvapi/include/public/krb5error-table.h
index e273c7adf..5ad8aa9c8 100644
--- a/lwadvapi/include/public/krb5error-table.h
+++ b/lwadvapi/include/public/krb5error-table.h
@@ -51,7 +51,7 @@ static const struct krb5err_lwerr krb5err_lwerr_map[] = {
     LWERR(ASN1_BAD_FORMAT, LW_ERROR_KRB5_ASN1_BAD_FORMAT),
     LWERR(ASN1_PARSE_ERROR, LW_ERROR_KRB5_ASN1_PARSE_ERROR),
     LWERR(ASN1_BAD_GMTIME, LW_ERROR_KRB5_ASN1_BAD_GMTIME),
-    LWERR(ASN1_MISMATCH_INDEF, LW_ERROR_KRB5_ASN1_MISMATCH_INDEF),
+    LWERR(ASN1_INDEF, LW_ERROR_KRB5_ASN1_INDEF),
     LWERR(ASN1_MISSING_EOC, LW_ERROR_KRB5_ASN1_MISSING_EOC),
 
     LWERR(KRB524_KRB4_DISABLED, LW_ERROR_KRB524_KRB4_DISABLED),
diff --git a/lwadvapi/include/public/lwerror-table-krb5.h b/lwadvapi/include/public/lwerror-table-krb5.h
index ca2b89ad8..b50ec1bba 100644
--- a/lwadvapi/include/public/lwerror-table-krb5.h
+++ b/lwadvapi/include/public/lwerror-table-krb5.h
@@ -15,7 +15,7 @@ S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_BAD_LENGTH, -1, "ASN.1 lengt
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_BAD_FORMAT, -1, "ASN.1 badly-formatted encoding" )
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_PARSE_ERROR, -1, "ASN.1 parse error" )
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_BAD_GMTIME, -1, "ASN.1 bad return from gmtime" )
-S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_MISMATCH_INDEF, -1, "ASN.1 non-constructed indefinite encoding" )
+S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_INDEF, -1, "ASN.1 non-constructed indefinite encoding" )
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5_ASN1_MISSING_EOC, -1, "ASN.1 missing expected EOC" )
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB524_KRB4_DISABLED, -1, "Kerberos version 4 support is disabled" )
 S ( LW_STATUS_MESSAGE_NOT_FOUND, LW_ERROR_KRB5KDC_ERR_NONE, -1, "No error" )
diff --git a/lwadvapi/include/public/lwerror.h b/lwadvapi/include/public/lwerror.h
index d2ce75084..e4a0e48d3 100644
--- a/lwadvapi/include/public/lwerror.h
+++ b/lwadvapi/include/public/lwerror.h
@@ -438,7 +438,7 @@
 #define LW_ERROR_KRB5_ASN1_BAD_FORMAT                      41709
 #define LW_ERROR_KRB5_ASN1_PARSE_ERROR                     41710
 #define LW_ERROR_KRB5_ASN1_BAD_GMTIME                      41711
-#define LW_ERROR_KRB5_ASN1_MISMATCH_INDEF                  41712
+#define LW_ERROR_KRB5_ASN1_INDEF                           41712
 #define LW_ERROR_KRB5_ASN1_MISSING_EOC                     41713
 
 /* From krb5/src/lib/krb5/error_tables/krb524_err.et */
